RUFAS/routines/field/manager/field_manager.py

```
@@ -394,37 +394,31 @@ def _setup_soil(soil_configuration: str, field_size: float) -> Soil:
 
         """
         soil_configuration_data = im.get_data(soil_configuration)
-        sand_content = soil_configuration_data["sand"]
-        silt_content = soil_configuration_data["silt"]
         residue = soil_configuration_data["initial_residue"]
-        nitrogen_mineralization_rate = soil_configuration_data["fresh_N_mineral_rate"]
-
-        soil_layers_config = list(soil_configuration_data.get("soil_layers").values())
+        soil_layers_config = soil_configuration_data.get("soil_layers")
         soil_layers_config.sort(key=lambda x: x.get("bottom_depth"))
```

We probably need to handle the case where we don't have the 'soil_layers' key because None would be returned in that case.

---

It is a critical input, so `InputManager` should fail if it is not present but this is still a good idea.